### PR TITLE
Add README and dotenv template file

### PR DESCRIPTION
**Pull request by ajhalili2006**
## In this PR
* `README` file - Includes commands reference and setup instructions
* The `.env` template file (`.env.sample`) - Boilerplate to placing instance's secrets and easy-to-follow template for people who want to set up their instances.

## Where I edited these files?
[Gitpod](https://gitpod.io), without any configuration or Dockerfile bull****.

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## To run the server, you need a Telegram Bot API token from BotFather.
```
Comment (vrumger) [inline .env.sample]: One `#` is enough.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+> :warning: **Make sure your user ID is included in `admins.json`!** While there's no
+way to add yourself as a operator in the `admins.json`, its required to ad yourself to the list,
+ensuring that you don't lose control to your gbanlist.
+4. Finally, fire up the console and type `npm start` to hit the road.
+
+## Commands Reference
+| Command | Permissions | Description |
+| ----- | ----- | ----- |
+| `/start` | _Everyone_ | Checks if the bot is dead. |
+| `/help` | _Everyone_ | This command doesn't exists. |
```
Comment (vrumger) [inline README.md]: If it doesn't exist, it shouldn't be on the list.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+| `/gban <userid/username/reply> <reason>`  | _Admins_ | Globally bans a user with a valid reason. |
+| `/ungban <userid/username/reply> <reason>`| _Admins_ | Globally unbans a user. |
```
Comment (vrumger) [inline README.md]: These commands don't support usernames.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+| `/stats` | _Admins_ | Pull global stats from bot database. |
```
Comment (vrumger) [inline README.md:21]: There's also `/stat <userid/reply>` for everyone.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+| `/clean` | _Admins_ | Cleans up the clutter collected from database.
+| `/newMembers` | _Admins_ | Checks and pulls new members from database. |
```
Comment (vrumger) [inline README.md]: This doesn't exist.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+// Not like this crap.
+// And there's no "operator" key in admins.json
+{
+    "admins": [ 172811422, 705519392 ],
+    "operator": 234480941
+}
+```
+> :warning: **Make sure your user ID is included in `admins.json`!** While there's no
+way to add yourself as a operator in the `admins.json`, its required to ad yourself to the list,
+ensuring that you don't lose control to your gbanlist.
```
Comment (vrumger) [inline README.md]: This is unnecessary.

File: README.md (hunk 3 above)
Comment (vrumger) [inline README.md]: The reason is optional and should be `[reason]`.

**Review by vrumger — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (ajhalili2006) [reply README.md]: Why ban reasons are optional? Should we make it required by default?

Comment (vrumger) [reply README.md]: I prefer to have it optional sometimes.

File: README.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+1. Clone the Git repository to your machine then install dependencies.
+2. Edit the `.env.sample` file and save it as `.env`.
+3. Make a `admins.json` file with an array of Telegram user IDs of admins you want to use admin-only commands.
+4. Finally, fire up the console and type `npm start` to hit the road.
+
+## Commands Reference
+| Command | Permissions | Description |
+| ----- | ----- | ----- |
+| `/start` | _Everyone_ | Checks if the bot is dead. |
+| `/stat <userid/reply>` | _Everyone_ | Pull stats for a user from bot database. |
```
Comment (vrumger) [inline README.md]: Can you sort the commands by relevancy?

File: README.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
+# SpamBlockers
+
+## Requirements
+* Node.js (v12.x or later is recommended) with `npm` (latest version recommended)
+* MongoDB server
+* Telegram Bot API token from BotFather
+* Log chat (optional)
```
Comment (vrumger) [inline README.md]: This should say "Log channel".

**Review by vrumger — CHANGES_REQUESTED** [review]

File: README.md (hunk 8 above)
Comment (ajhalili2006) [reply README.md]: @YouTwitFace Why should I log bans/unbans in a channel?

File: README.md (hunk 7 above)
Comment (ajhalili2006) [reply README.md]: Okay, I'll queued that.

File: README.md (hunk 8 above)
Comment (vrumger) [reply README.md]: That's how it is in the `.env` and consistency is nice.

**Review by vrumger — APPROVED** [review]

File: README.md (hunk 8 above)
Comment (robbyoconnor) [reply README.md]: @YouTwitFace Every action should be audited.

Comment (AndreiJirohHaliliDev2006) [reply README.md]: @robbyoconnor I agreed too, @robbyoconnor.

Comment (AndreiJirohHaliliDev2006) [reply README.md]: @AndreiJirohHaliliDev2006 Transparency is the key to trust admins too.

**Pull request merged**